payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -180,5 +173,22 @@ internal abstract class BaseAddPaymentMethodFragment(
                 SupportedPaymentMethod.Card -> CardDataCollectionFragment::class.java
                 else -> ComposeFormDataCollectionFragment::class.java
             }
+
+        private val transformToPaymentMethodCreateParams = TransformToPaymentMethodCreateParams()
```

Comment:
Seems we won't need to create an instance if `TransformToPaymentMethodCreateParams#transform` is static

Reply:
yeah, there are several Transform classes that I have, so I think I would like to make a separate PR to address all of these.